test/net/sourceforge/kolmafia/request/LatteRequestTest.java

```diff
@@ -0,0 +1,35 @@
+package net.sourceforge.kolmafia.request;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class LatteRequestTest {
+  @BeforeEach
+  public void beforeEach() {
+    KoLCharacter.reset("LatteRequest");
+    Preferences.reset("LatteRequest");
+  }
+
+  private LatteRequestTest() {}
+  ;
+
+  @ParameterizedTest
+  @ValueSource(
```

Comment:
may as well do `CsvSource` with a custom delimiter (e.g. `|`, which you already have set up).